src/lib/tests/tests.cpp

```diff
@@ -1,6 +1,399 @@
+#include "simd/simd_doubles.h"
+#include "simd/simd_floats.h"
+#include "simd_config.h"
 #include "gtest/gtest.h"
 
-// a blank test
-TEST(DistopiaTests, BlankTest) {
-  SUCCEED();
+/* Section for testing SimdFloatX4 packed structs
+Notes
+* operates on __m128 types which require 16 bit alignment
+
+*/
+
+TEST(SimdFloatX4, SetAndStore) {
+  SimdFloatX4 fx4;
+  float *result = new float[4];
```

Comment:
How do you know that it’s unaligned? 😉

Reply:
true i should have used `alignof` or `aligned_alloc(15)` or something.